support-frontend/assets/components/ticker/contributionTicker.jsx

```diff
@@ -31,9 +32,11 @@ const getInitialTickerValues = (tickerJsonUrl: string): Promise<StateTypes> =>
       return { totalSoFar, goal };
     });
 
-const percentageTotalAsNegative = (total: number, goal: number) => {
+const percentageToTranslate = (total: number, goal: number, tickerType: TickerType) => {
   const percentage = ((total / goal) * 100) - 100;
-  return percentage > 0 ? 0 : percentage;
+  const endOfFillPercentage = tickerType === 'unlimited' ? -15 : 0;
+
+  return percentage > 0 ? endOfFillPercentage : percentage;
```

Comment:
Maybe a comment here explaining what percentage > 0 means (or put it into a well-named variable)

Reply:
👍